basics/patching.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: contribute
+title: Patch acceptance process
+---
+
+# Patch Acceptance Process
+
+1. Read the [Bazel governance plan](/governance.html).
+1. Discuss your plan and design, and get agreement on a GitHub issue or our
+   [mailing list](https://groups.google.com/forum/#!forum/bazel-dev).
+1. PRs that change or add behavior are not accepted without being tied to an
+   issue.
+1. Significant changes need a [design document](/designs/index.html).
```

Comment:
If you're proposing significant changes, write a [design document](/designs/index.html).

(Do we have a link to what "significant changes" mean?)

Reply:
Done.